startupmanager/src/main/java/org/apache/sling/launchpad/startupmanager/Activator.java

```diff
@@ -37,40 +39,30 @@
 
     private static final Logger log = LoggerFactory.getLogger(Activator.class);
 
-    private ServiceRegistration<?> mbeanServerReg;
-
-    private ServiceRegistration<StartupListener> mbeanStartupListenerReg;
-
     private StartupListenerTracker startupListenerTracker;
 
     @Override
     public void start(final BundleContext bundleContext) {
         startupListenerTracker = new StartupListenerTracker(bundleContext);
-        mbeanServerReg = registerMBeanServer(bundleContext);
```

Comment:
Don't we need to unregister all the created services?

Reply:
The OSGi framework will automatically unregister services for a bundle when it is stopped, or do you think we should unregister it at a different point in time?

Reply:
No, at the stop. I didn't know this, thanks!